colorfield/widgets.py

```diff
@@ -27,10 +30,22 @@ def get_context(self, name, value, attrs=None):
                 "widget": self,
                 "name": name,
                 "value": value,
+                # ensure that there is an id
+                "data_coloris_id": "coloris-"
+                + context.get("id", get_random_string(length=32)),
```

Comment:
Nice fallback!

Reply:
The default value is already alphanumeric, with upper and lower case